### ISSUE TITLE
Add export function

### ISSUE THREAD

**Issue by vallezw** [SheetAble/SheetAble#28]
Adding an export function. To export all sheets in a zip file.

**Comment by chagz13** [comment]
Very good development

**Comment by a2brew** [comment]
This would be a great feature! But also maybe add a database/config export feature as a part of this? So that way a user/admin can restore an instance via the UI, if say the Docker config doesn't load properly? Just an idea!

**Comment by vallezw** [comment]
Agreed, would be kind of nice! I'll try to implement this in the new version @a2brew
